tests/tutorials/test_dependencies.py

```diff
@@ -0,0 +1,53 @@
+from typing import TYPE_CHECKING
+import re
+from pathlib import Path
+
+import pytest
+
+if TYPE_CHECKING:
+    from pytest_virtualenv import VirtualEnv
+
+try:
+    from docs.source.utils.notebook import InstallationCell
+except ImportError:
+    pytest.skip(reason="`doc` is not available", allow_module_level=True)
```

Comment:
This doesn't seem like a good solution too: we call something a test - and then we request for it some random dependencies and if they are not met - just skip it.

Reply:
Removed the skipping.